internal/report/prometheus.go

```diff
@@ -9,22 +9,33 @@ const namespace = "popeye"
 
 // Metrics
 var (
-	score = prometheus.NewGauge(prometheus.GaugeOpts{
+	score = prometheus.NewGaugeVec(prometheus.GaugeOpts{
 		Namespace: namespace,
 		Name:      "score",
 		Help:      "Score of kubernetes cluster.",
-	})
-	grade = prometheus.NewGauge(prometheus.GaugeOpts{
+	},
+		[]string{
+			"cluster",
+			"namespace",
+		})
+	grade = prometheus.NewGaugeVec(prometheus.GaugeOpts{
 		Namespace: namespace,
 		Name:      "grade",
 		Help:      "Grade of kubernetes cluster. (1: A, 2: B, 3: C, 4: D, 5: E, 6: F)",
-	})
```

Comment:
@eminugurkenar I think we could use a helper here for this type of gauge since we are repeating ourselves. Something like makeGauge(name, help string) prometheus.GaugeVec. Also might as well change `title` to resource since this is what the tag represents?

Lastly I am still of the opinion that we should roll all the levels into a single metric with a level tag. ie `popeye_sanitizers_total{cluster,  job, level, namespace, resource}. My take here is at the end of the day these metrics are not really user consumables but will feed other subsystems ie grafana or report generators. With a single series one could plot a sanitizer report aggregated by level tag and see a chart with info, warning, errors. I think it would be more usable that way. Do you feel that would make sense?

Reply:
title became resource.  For popeye_sanitizers_total , i think for "aggregation" yes you are right it is better to keep them as single metric so users create some notifications by just ingoring level Info  etc.. So converted them to single metric now.